var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -46,6 +49,7 @@ class Zee(CMakePackage):
 
     depends_on('metis+int64')
     depends_on('petsc +int64', when='+petsc')
+    depends_on('timemory', when='+timemory')
```

Comment:
Since you removed the variants, could you please add them to `deploy/config/packages.yaml`?

Reply:
Done!

Reply:
For what is exactly `deploy/configs/packages.yaml` meant for @matz-e ? Because I am not sure we should add timemory in there.

The original `~caliper~cuda~cupti~gperftools~papi~python~mpi` variants in `zee/package.py`  was just to prevent @WeinaJi to build a minimal version of timemory on her workstation. But there is already a timemory module on BB5 which provides everything:

```
git grep timemory deploy/packages | cat
deploy/packages/external-libraries.yaml:      - timemory+mpi+cuda+cupti+caliper~gperftools~python@3.0.0a
```

Reply:
> For what is exactly `deploy/configs/packages.yaml` meant for @matz-e ? Because I am not sure we should add timemory in there.
> 
> The original `~caliper~cuda~cupti~gperftools~papi~python~mpi` variants in `zee/package.py` was just to prevent @WeinaJi to build a minimal version of timemory on her workstation. But there is already a timemory module on BB5 which provides everything:
> 
> ```
> git grep timemory deploy/packages | cat
> deploy/packages/external-libraries.yaml:      - timemory+mpi+cuda+cupti+caliper~gperftools~python@3.0.0a
> ```

Good catch. It's just to prevent `timemory` from rebuilding needlessly. From what I grasp, it's not that simple to build?

@WeinaJi sorry, I guess those flags should then maybe go into the Ubuntu `packages.yaml` and the one in `deploy` should match the variants @tristan0x listed.

Reply:
> Good catch. It's just to prevent `timemory` from rebuilding needlessly. From what I grasp, it's not that simple to build?

The package is actually huge and support many tools.

> @WeinaJi sorry, I guess those flags should then maybe go into the Ubuntu packages.yaml and the one in deploy should match the variants @tristan0x listed.

comment applied on d896695b996